server/lib/nl/fulfillment/base.py

```diff
@@ -249,3 +253,38 @@ def _svg_or_topic_to_svs(state: PopulateState, sv: str,
     return charts
 
   return []
+
+
+def open_topics_ordered(svs: List[str]) -> List[str]:
+  opened_svs = []
+  sv_set = set()
+  for rank, sv in enumerate(svs):
+    for sv in _open_topic_in_var(sv, rank):
+      if sv not in sv_set:
+        opened_svs.append(sv)
+        sv_set.add(sv)
+  return opened_svs
+
+
+def _open_topic_in_var(sv: str, rank: int) -> List[str]:
+  if utils.is_sv(sv):
+    return [sv]
+  if utils.is_topic(sv):
```

Comment:
Why is nothing being returned from this conditional path?

Reply:
Great eyes!  I just fixed the bug.